src/pharmpy/plugins/nlmixr/model.py

```diff
@@ -62,15 +70,35 @@ def convert_model(model):
         description=model.description,
     )
 
-    # Update dataset to lowercase and add evid
-    nlmixr_model = modify_dataset(nlmixr_model)
+    # Update dataset
+    if model.dataset is not None or len(model.dataset) != 0:
+        if keep_etas is True:
+            nlmixr_model = nlmixr_model.replace(
+                modelfit_results=ModelfitResults(
+                    individual_estimates=model.modelfit_results.individual_estimates
+                )
+            )
+        nlmixr_model = translate_nmtran_time(nlmixr_model)
+        # FIXME: dropping columns runs update source which becomes redundant.
+        # drop_dropped_columns(nlmixr_model)
+        if all(x in nlmixr_model.dataset.columns for x in ["RATE", "DUR"]):
+            nlmixr_model = drop_columns(nlmixr_model, ["DUR"])
+        nlmixr_model = nlmixr_model.replace(
+            datainfo=nlmixr_model.datainfo.replace(path=None),
+            dataset=nlmixr_model.dataset.reset_index(drop=True),
+        )
+    else:
+        print_warning("No connected dataset or dataset is empty")
```

Comment:
No need for warning here. This is a legitimate case

Reply:
Removed warning